### PR TITLE
Changes the PR template.

### PR DESCRIPTION
**Pull request by AnCopper**
# Document the changes in your pull request.

Revises PR template to adding a testing and issue closer section.

Maintainability starts with the PR.

# Explain why it's good for the game.

Not user facing.

QOL for longer term DevOps. When dealing with unexpected behaviors in the wild, knowing if it is intended can be helpful in some cases.
Moreover, it provides a place for testing/images to be present without getting lost in long lines of commits/comments/etc.

This would also allow for a seamless way to keep a one stop place for documentation in cases of larger PRs.

# Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

</details>

# Changelog

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,22 +1,39 @@
...
 
-<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
+<!-- Remove this text and explain what the purpose of your PR is.
 
-## Why It's Good For The Game
+Mention if you have tested your changes. If you changed a map, make sure you used the mapmerge tool.
+If this is an Issue Correction, you can type "Fixes Issue #169420" to link the PR to the corresponding Issue number #169420.
+
+Prefix the PR title with [admin] if it involves something admin related.
+Prefix the PR title with [s] if you are fixing an exploit, so that it is not announced on the Yogstation Discord and the server.
```
Comment (morrowwolf) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: bro, at least check your copy pastes.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,22 +1,39 @@
...
+Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.
+
+</details>
+
+
+# Changelog
 
-<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->
+<!-- Edit the changelog below to reflect the changes made by this PR, even if the changes are minor - required for every PR.
+If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name. Be sure to properly mark your PRs to prevent unnecessary GBP loss. Maintainers freely reserve the right to remove and add tags should they deem it appropriate. -->
```
Comment (morrowwolf) [inline .github/PULL_REQUEST_TEMPLATE.md:33]: This is mandated for CM. You must add your CKEY after the CL if your github name doesn't match.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,22 +1,39 @@
 <!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
 
-## About The Pull Request
+# Document the changes in your pull request.
```
Comment (morrowwolf) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: Keep this as "about the pull request"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,22 +1,39 @@
...
+
+Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.
+
+</details>
+
+
+# Changelog
 
-<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->
+<!-- Edit the changelog below to reflect the changes made by this PR, even if the changes are minor - required for every PR.
```
Comment (morrowwolf) [inline .github/PULL_REQUEST_TEMPLATE.md:32]: Keep the original line here

**Review by morrowwolf — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -1,22 +1,39 @@
...
 
-<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
+<!-- Remove this text and explain what the purpose of your PR is.
 
-## Why It's Good For The Game
+Mention if you have tested your changes. If you changed a map, make sure you used the mapmerge tool.
+If this is an Issue Correction, you can type "Fixes Issue #169420" to link the PR to the corresponding Issue number #169420.
+
+Prefix the PR title with [admin] if it involves something admin related.
+Prefix the PR title with [s] if you are fixing an exploit, so that it is not announced on the CMSS13 Discord and the server.
```
Comment (fira) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: Do we actually have this in our tooling or is this just copypasting?

**Review by fira — COMMENTED** [review]
I just lost 3 days worth of zen budget reading non-sentence titles inconsistently ended with full stops.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (morrowwolf) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: Pretty sure it's just copy pasta.

**Comment by Unknownity** [comment]
Changelog is not required, no player facing changes.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -1,22 +1,36 @@
...
+# About the pull request
 
-<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
+<!-- Remove this text and explain what the purpose of your PR is.
 
-## Why It's Good For The Game
+Mention if you have tested your changes. If you changed a map, make sure you used the mapmerge tool.
+If this is an Issue Correction, you can type "Fixes Issue #169420" to link the PR to the corresponding Issue number #169420.
+
+Remember: something that is self-evident to you might not be to others. Explain your rationale fully, even if you feel it goes without saying.-->
```
Comment (harryob) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
Remember: something that is self-evident to you might not be to others. Explain your rationale fully, even if you feel it goes without saying. -->
```

**Review by harryob — CHANGES_REQUESTED** [review]

**Comment by harryob** [comment]
if this bugs someone enough they can make a new new PR template but i like this and i like mandated testing

**Pull request merged**